deepmd/pt/model/atomic_model/linear_atomic_model.py

```diff
@@ -185,10 +188,12 @@ def forward_atomic(
         ener_list = []
 
         for i, model in enumerate(self.models):
+            ori_map = model.get_type_map()
+            updated_atype = self.remap_atype(extended_atype, ori_map, self.type_map)
```

Comment:
the map should be prepared and upload to GPU at `__init__`, so we can avoid host-GPU data transfer in forward.